tests/integration/test_storage.sh

```diff
@@ -69,20 +69,20 @@ trap teardown EXIT
 # If /etc/sysconfig/docker-storage-setup is missing, atomic should create the file.
 
 rm -f /etc/sysconfig/docker-storage-setup
-OUTPUT=$(${ATOMIC} storage modify --add-device $TEST_DEV_1 2>&1)
+if [ -z "$VGROUP" ];then
+   VGROUP="test-atomic-storage"
+   OUTPUT=$(${ATOMIC} storage modify --add-device $TEST_DEV_1 --vgroup $VGROUP 2>&1)
```

Comment:
`Why did you change the tests as part of this PR beyond just reverting #915? Can we leave that out and worry about it in a separate PR?`

Because the tests started failing. Was this environment (fedora/25/cloud) recently introduced in the CI? Not sure how the existing test would pass on this environment. Since the test will try to find the volume group backing the rootfilesystem, and if it's not present it will look for `VG=` which was not defined in the exisiting tests. So d-s-s would fail. Obviously just setting VG won't work since docker is already set to run on loopback devices. We need to clear that first.

I believe `atomic storage reset` could come handy here like you mentioned.